src/main/java/uk/gov/hmcts/darts/authorisation/exception/AuthorisationError.java

```diff
@@ -0,0 +1,54 @@
+package uk.gov.hmcts.darts.authorisation.exception;
+
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import uk.gov.hmcts.darts.common.exception.DartsApiError;
+
+@Getter
+@RequiredArgsConstructor
+public enum AuthorisationError implements DartsApiError {
+
+    USER_NOT_AUTHORISED_FOR_COURTHOUSE(
```

Comment:
Again could be split as per my comments above.

Reply:
@hemantasharma1129 advises to keep it for now. He wanted the same Unauthorised response for all Controller operations with the custom 'Authorisation' annotation across 5 contexts with a contextId and securityRoles parameters.